tests/telemetry/test_telemetry.py

```diff
@@ -19,14 +21,28 @@ def get_list_stdout(cmd_out):
         out_list.append(result)
     return out_list
 
+def restart_telemetry(duthost):
+    """ Restart telemetry process
+    """
+    duthost.shell('sudo config feature telemetry disabled', module_ignore_errors=False)['stdout_lines']
```

Comment:
remove 'stdout_lines' since you are not capturing any output

Reply:
This is not resolved.

Reply:
removed this code completely.